core/domain/email_manager.py

```diff
@@ -364,9 +365,9 @@ def send_mail_to_admin(email_subject, email_body):
 
     app_id = app_identity_services.get_application_id()
     body = '(Sent from %s)\n\n%s' % (app_id, email_body)
-
+    system_name_email = '%s <%s>' % ('SYSTEM', feconf.SYSTEM_EMAIL_ADDRESS)
```

Comment:
Using literal constants is discouraged, especially if the usage is repeated. Move the constant to feconf as described above.

Reply:
On it!

Reply:
Done